server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -180,21 +177,13 @@ private ConfigProperties() {
         DB_PASSWORD,
     };
 
-    // Pinsetter
-    public static final String TASKS = "pinsetter.tasks";
-    public static final String DEFAULT_TASKS = "pinsetter.default_tasks";
-    public static final String ENABLE_PINSETTER = "candlepin.pinsetter.enable";
-    public static final String PINSETTER_ASYNC_JOB_TIMEOUT = "pinsetter.waiting.timeout.seconds";
-    public static final String PINSETTER_MAX_RETRIES = "pinsetter.retries.max";
-    public static final int PINSETTER_MAX_RETRIES_DEFAULT = 10;
-
     public static final String[] DEFAULT_TASK_LIST = new String[] {};
```

Comment:
This can also be removed. It was the default task list of the old pinsetter jobs, which we removed one by one as we ported jobs.

Reply:
done